src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -32,6 +33,33 @@ export const stickyNoteMeasures = {
   width: 29.8, /* to allow text to fit in one line with tooltip */
 };
 
+const printStyles = {
+  modal: css`
+    position: relative;
```

Comment:
if you moved these styles to the popup body, could you ignore the modal?

Reply:
I don't think so. It needs the position changed so it the whole content of highlights is being printed, not only one page.